test/processors/concerns/http_files_test.rb

```diff
@@ -45,9 +43,9 @@ def logger
   end
 
   it 'can download from https url' do
+    fi = File.open(in_file('test_short.mp2'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.